components/Modals/Modals.styled.js

```diff
@@ -0,0 +1,88 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export const StyledModal = styled.div`
+  position: fixed;
+  inset: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.5);
```

Comment:
You should really consider defining your colors in the same way every time. At the moment, you're using RGBA, hash values, and plain text. I would recommend setting your colors as root in styles.js and then calling them with `var`. This will also make your code more readable for any devs who come after you! 😄